src/NHSD.BuyingCatalogue.Ordering.Services/OrderService.cs

```diff
@@ -71,12 +71,6 @@ public async Task<Order> GetOrderCompletedStatus(CallOffId callOffId)
                 .SingleOrDefaultAsync();
         }
 
-        [ItemCanBeNull]
```

Comment:
I think we can remove the `using JetBrains.Annotations;` statement now you have removed the annotation.

Reply:
Removed `using JetBrains.Annotations;`